src/ASM/ASMs2Dmx.C

```diff
@@ -277,60 +278,59 @@ bool ASMs2Dmx::generateFEMTopology ()
 
   size_t iel = 0, inod = 0;
   for (auto& it : m_basis)
-    for (i2 = 0; i2 < it->numCoefs_v(); i2++)
-      for (i1 = 0; i1 < it->numCoefs_u(); i1++)
+    for (int i2 = 0; i2 < it->numCoefs_v(); i2++)
+      for (int i1 = 0; i1 < it->numCoefs_u(); i1++)
       {
         myNodeInd[inod].I = i1;
         myNodeInd[inod].J = i2;
         myMLGN[inod++]    = ++gNod;
       }
 
-  int firstItgNode = this->getFirstItgElmNode();
-  int nElmNode = std::accumulate(elem_size.begin(), elem_size.end(), 0);
+  const size_t firstItgElmNode = this->getFirstItgElmNode();
+  const size_t nElmNode = std::accumulate(elem_size.begin(), elem_size.end(), 0);
```

Comment:
well, the normal reasons, const helps avoid mistakes and it aids the compiler by telling it that it can put this in a register. i figured i could add since i updated the types anyways, but i'm not insisting.

Reply:
no, by all means, keep them if there is a reason (and helps the compiler).